internal/output/outputter.go

```diff
@@ -124,6 +116,12 @@ func (o *Outputter) StartBuildStep(format string, a ...interface{}) {
 	o.curBuildStep++
 }
 
+func (o *Outputter) Summary(format string, a ...interface{}) {
+	o.logger.Infof("%s", o.blue().Sprint("──┤ Status ├──────────────────────────────────────────────────────────"))
```

Comment:
Per our discussion about how it's confusing to show all services when not all of them have updated...I changed the summary section title to "Status." In my mind that helps a little bit, but I'm not sure if this is a distinction that makes sense to others.